### PR TITLE
Stop running hooks on pr merge

### PR DESCRIPTION
**Pull request by zeripath**
Alternative fix for #6946 

This just removes the final SSH_ORIGINAL_COMMAND preventing the hooks from being run whilst I figure out how to fix the issues in #6961

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

**Review by mrsdizzie — APPROVED** [review]

**Comment by zeripath** [comment]
Hmm... the new unit test failure is very interesting. It doesn't appear to have anything to do with my changes.

**Comment by zeripath** [comment]
In particular the failure is at:

```
panic: Failed to execute 'git config --global core.quotepath false': error: could not lock config file /root/.gitconfig: File exists
29
```

Has there been a change of the docker environment?

File: models/pull.go [hunk 1]
```diff
@@ -556,6 +556,8 @@ func (pr *PullRequest) Merge(doer *User, baseGitRepo *git.Repository, mergeStyle
 	}
 
 	env := PushingEnvironment(doer, pr.BaseRepo)
+	// FIXME: #6946 requires protected branches to be ignored
+	env = env[:len(env)-1]
```
Comment (lunny) [inline models/pull.go]: This will assume that env is the last one. It's danger. I would like to traverse all the envs and remove the extract one.

**Review by lunny — CHANGES_REQUESTED** [review]

File: models/pull.go (hunk 1 above)
Comment (zeripath) [reply models/pull.go]: It's not a longstanding fix it's just meant to last until I get #6961 fixed.

**Comment by zeripath** [comment]
OK I've just removed the SSH_ORIGINAL_COMMAND from the pushing environment as I suspect that it's not working correctly in any case.

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
But `repo.OwnerName` should still be `repo.MustOwnerName()`?

**Comment by zeripath** [comment]
Yeah, any comprehensive fix will need that - could add it now if preferred.

**Pull request merged**